README.rst

```diff
@@ -85,6 +85,11 @@ Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 
 Changes
 =======
+v0.4.1
```

Comment:
Given that 0.4.0 has not been released, there is no need for another version change. Just update the change log for 0.4.0.

Reply:
0.4.0 has been released. Maybe I should have waited though and incorporated this with that. Lesson learned for next time.